### ISSUE TITLE
Fix TAG ?= bug in v1.2

### ISSUE THREAD

**Issue by jackgr** [kubernetes/deployment-manager#348]
The file `include.mk` at the top level in release v1.2 contains the following line:

TAG ?= $(shell echo `date +"%s"`_`date +"%N"`)

This line causes the TAG to change every time the build is run, which results in an error when one or more of the docker images are pushed:

```
Successfully built 2cbab82285a8
gcloud docker push gcr.io/fractal-dev/expandybird:1457382911_763335024
The push refers to a repository [gcr.io/fractal-dev/expandybird] (len: 1)
error getting tags for gcr.io/fractal-dev/expandybird: Tag does not exist for 1457382911_763335024
```

**Comment by jackgr** [comment]
When the bug is fixed, we should cut a new label (e.g., v1.2.1) that includes the fix, and update all files that contain the label.

**Comment by jackgr** [comment]
Closed by #351.
